.pre-commit-config.yaml

```diff
@@ -13,6 +13,4 @@ repos:
     rev: current
     hooks:
     -   id: check-bash-syntax
-    -   id: check-jenkins-pipelines
-    -   id: check-jjbb
```

Comment:
These checks are not really something to be linted when moving to GitHub actions, so this should help us to avoid the need to use the internal docker registry.